Cannot find module 'word-wrap'

```
Oops! Something went wrong! :(

ESLint: 8.43.0

Error: Cannot find module 'word-wrap'
Require stack:
- /path/to/node_modules/optionator/lib/help.js
- /path/to/node_modules/optionator/lib/index.js
- /path/to/node_modules/eslint/lib/options.js
```

--- a/src/help.ls
+++ b/src/help.ls
@@ -1,6 +1,6 @@
 {id, find, sort, min, max, map, unlines} = require 'prelude-ls'
 {name-to-raw, dasherize, natural-join} = require './util'
-word-wrap = require 'word-wrap'
+word-wrap = require '@aashutoshrathi/word-wrap'
 
 wordwrap = (a, b) ->
   [indent, width] = if b == undefined then

